test/master.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const metatests = require('metatests');
+const { Worker } = require('worker_threads');
+
+metatests.test(
+  'Start server without balancer with 3 workers 2 ports',
+  (test) => {
+    const workersNum = 3;
+
+    for (let i = 0; i < workersNum; i++) {
```

Comment:
```suggestion
    for (let i = 1; i < workersNum; i++) {
```
- Number of workers should be equal to number of ports
- Thread 0 in intended for system use, so we will skip it